apps/challenges/views.py

```diff
@@ -245,6 +246,31 @@ def challenge_detail(request, challenge_host_team_pk, challenge_pk):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+@api_view(['GET'])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def team_name_for_challenge(request, challenge_pk):
+    """
+    Returns the name of the user's participation team in the current challenge.
```

Comment:
@Kajol-Kumari can you please add appropriate docstrings for the API. See this for an example https://github.com/Cloud-CV/EvalAI/blob/master/apps/jobs/views.py#L1986.